beangulp/__init__.py

```diff
@@ -122,8 +122,10 @@ def _extract(ctx, src, output, existing, reverse, failfast, quiet):
               help='Just print where the files would be moved.')
 @click.option('--failfast', '-x', is_flag=True,
               help='Stop processing at the first error.')
+@click.option('--quiet', '-q', count=True,
+              help="Suppress all output.")
```

Comment:
```suggestion
              help='Suppress all output.')
```
The code you copied also uses the wrong quote style. You can add a patch to fix that code as well, if you want.